.github/workflows/publish.yml

```
@@ -29,14 +29,17 @@ jobs:
 
       - name: Build
         run: yarn build
+
+      - name: Run tests
+        run: yarn run-instance
       
       - name: Write release version
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:15: Double quote to prevent globbing and word splitting

(shellcheck)